apps/website/src/server/trpc/router/admin/show-and-tell.ts

```diff
@@ -29,6 +33,54 @@ export const adminShowAndTellRouter = router({
     .input(z.string().cuid())
     .query(({ input }) => getPostById(input)),
 
+  getModComments: permittedProcedure
+    .input(z.string().cuid())
+    .query(async ({ input }) => {
+      const comments = await getModCommentsForEntry(input);
+      if (!comments) {
+        throw new TRPCError({
+          code: "NOT_FOUND",
+          message: "Comments not found",
+        });
+      }
+      return comments;
+    }),
+
+  addModComment: permittedProcedure
+    .input(
+      z.object({
+        entryId: z.string().cuid(),
+        modId: z.string().cuid(),
+        comment: z.string(),
+        isInternal: z.boolean(),
+      }),
+    )
+    .mutation(async ({ input }) => {
+      return await addModComment(
+        input.entryId,
+        input.modId,
+        input.comment,
+        input.isInternal,
+      );
+    }),
```

Comment:
Nit pick but we can spare one asyc/await promise resolving by just returning the promise directly here.

```suggestion
    .mutation(({ input }) => addModComment(
      input.entryId,
      input.modId,
      input.comment,
      input.isInternal,
    )),
```